bridge/client/app/_components/ktb-deployment-list/ktb-deployment-list.component.ts

```diff
@@ -61,43 +60,56 @@ export class KtbDeploymentListComponent implements OnInit, OnDestroy {
   constructor(public _changeDetectorRef: ChangeDetectorRef, private route: ActivatedRoute, private dataService: DataService, private router: Router, private location: Location) { }
 
   ngOnInit(): void {
-    this.route.params
+    this.dataService._remediationsUpdated
```

Comment:
I see your confusion, but I would not change that behaviour currently, since the project is the parent of all those elements. The only thing that we should consider is, that we should probably also reload the project periodically, since there might be changes (stage configuration, etc.) or allow a manual reload from the page, without having to reload the Bridge.

However, I would just allow components to also retrieve remediations, roots, etc. via the Observables, without subscribing for the project, but basically I would keep this structure.

Reply:
The project is updated now every 30 seconds in the service screen. But we should also consider updating it in the environment screen. I would not update it in all views, because it is not needed in the sequence screen for example

Reply:
There is already an issue for that I am already working on https://github.com/keptn/keptn/issues/4048